docs/src/test-fixtures-js.md

```diff
@@ -675,27 +675,30 @@ test('passes', async ({ database, page, a11y }) => {
 
 ## Box fixtures
 
-You can minimize the fixture exposure to the reporters UI and error messages via boxing it:
+Usually, custom fixtures are reported as separate steps in in the UI mode, Trace Viewer and various test reports. They also appear in error messages from the test runner. You can minimize the fixture exposure in the UI by "boxing" it.
 
 ```js
 import { test as base } from '@playwright/test';
 
 export const test = base.extend({
-  _helperFixture: [async ({}, use, testInfo) => {
+  helperFixture: [async ({}, use, testInfo) => {
+    // ...
   }, { box: true }],
 });
 ```
 
+This is useful for non-interesting helper fixtures. For example, an auto fixture that sets up some common data can be safely hidden from a test report.
```

Comment:
```suggestion
This is useful for non-interesting helper fixtures. For example, an [automatic](./test-fixtures.md#automatic-fixtures) fixture that sets up some common data can be safely hidden from a test report.
```

When I googled for "auto fixture playwright", I didn't really find anything. The name in the docs is "automatic fixture", so let's also use that here. And a link so I don't have to google next time :D

<img width="879" alt="Screenshot 2024-07-26 at 16 00 10" src="https://github.com/user-attachments/assets/2e3f09a4-3909-4d6e-a100-87ef3150cb2d">

The google results for "automatic fixture playwright" aren't a lot better, which suggests that some work on SEO is due